content/projects/cannerio.md

```diff
@@ -1,22 +1,86 @@
+startertemplaterepo: Canner/canner-demo
 images:
-  - path: /images/canner-dashboard.png
+  - path: /images/canner-1.png
+  - path: /images/canner-2.png
+  - path: /images/canner-3.png
 ---
 ## CannerIO CMS
 
-Canner CMS is the next generation CMS platform, we help you build your projects faster with beautiful CMS dashboard. With Canner CMS you are able to create any CMS websites using Canner schema.  Canner have supported over 30+ component options [Learn more](https://docs.canner.io/en/plugins/introduction).  We not only support CMS but also a complete tool sets for your work flow, such as Image server, hosting, API, cloud functions, access management, CLI, integrate with firebase, and more. [more information please visit our website](https://www.canner.io)
+> Canner is a universal Content Management System (CMS) framework that allows you to build CMS in React JSX(XML-like) for Firebase, GraphQL, Restful API, Prisma. In other words, an agnostic CMS framework for any applications and data sources. 
 
-## Starter Template
+![Canner flow](https://cdn.canner.io/images/home/flow.gif)
 
-Here is a starter template [app-demo](https://github.com/Canner/app-demo), and follow step-by-step tutorial [here](https://docs.canner.io/en/get-started).
+## Benefits and advantages
 
-See more demos [HTML5UP websites with CMS on Canner](https://github.com/Canner/canner-html5up-demo)
+### 🌍 Universal CMS framework
+
+Canner is flexible and agnostic, that you can ***learn once and create any form of CMS for many different databases.***
+
+### ⛄ Create data model and UI design at the same time.
+
+By defining a Canner schema in JSX, you can determine how the CMS stores data in your databases and the appearance of the CMS UI/UX.
+
+![schema](https://cdn.canner.io/images/home/schema.png)
+
+### 👩‍💻 One schema to any database
+
+With different connectors, you are able to create a CMS to interact with ***Firebase, MySQL, PostgreSQL, GraphQL with the same schema***.
+
+### 😎 Easy to maintain and extend
+
+Since Canner schema ***is only a JSX (XML-like) language without any logistic code***, in other words, you can maintain your code in a configuration file and easy to maintain using version controls.
+
+
+## 🙌 Why create Canner?
+
+Before we built Canner CMS framework, we've tried other CMS frameworks such as Wordpress, Drupal, etc. What we found was that getting simple functions to work was easy enough, but once you try to build and scale into your product or your service you ran into deeper issues...
+
+1. **Humongous codebase:**  Each CMS framework is humongous and complex, if you want to use them you have to install a huge codebase into your system, and most of the time it'll affect and mess with all your existing code.
+
+> Canner's CMS is only a React component.
+
+2. **Hardcoded and limited APIs:**  APIs should not be predefined and even hard coded into the codebase, which is critical for building advanced use cases and impossible to build into your system.
```

Comment:
These numbered list items are all coming out as `1` - maybe try removing the newline between them?

https://deploy-preview-98--headlesscms.netlify.com/projects/canner